Gifski/AppDelegate.swift

```diff
@@ -300,14 +298,16 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		circularProgress.progress = 0
 		circularProgress.animated = true
 
-		progress = gifski.convertFile(
+        progress = Progress(totalUnitCount: 1)
+        progress?.becomeCurrent(withPendingUnitCount: 1)
+		gifski.convertFile(
 			inputUrl,
 			outputUrl: outputUrl,
 			quality: defaults["outputQuality"] as! Double,
 			dimensions: choosenDimensions,
 			frameRate: choosenFrameRate
 		)
-		progress?.publish()
```

Comment:
With these changes, I no longer get the nice progress bar in Finder.

Here, I'm saving to `Downloads` (The progress bar only shows in `master`, not this PR):

<img width="106" alt="screen shot 2018-02-26 at 11 43 20" src="https://user-images.githubusercontent.com/170270/36653777-54364f28-1aea-11e8-88db-113a0ed3686b.png">

Reply:
Oh. I didn't see any effect and the docs don't help at all, but now I know where to look!
Will look into that and fix.. suspect it might have something to do with progress.fileurl

Reply:
I re-added publish/unpublish and the fileURL stuff.
The NSProgress header file comments had some more info regarding this, and the way you did it was correct..